### PR TITLE
Testing system

### PR DESCRIPTION
**Pull request by Stepania**
Prototype testing infrastructure added.

### CODE REVIEW

File: NutrientBalance.ipynb
Comment (HamishBrownPFR) [inline NutrientBalance.ipynb:1]: This one should not be changed, can you undo the change here

File: modelCsharp/SVSModel.sln
Comment (HamishBrownPFR) [inline modelCsharp/SVSModel.sln:1]: Can you confirm all the components added are necessary

File: modelCsharp/SVSModel/ClassDiagram1.cd
Comment (HamishBrownPFR) [inline modelCsharp/SVSModel/ClassDiagram1.cd:1]: Is this class needed

File: modelCsharp/SVSModel/Configuration/DummyChecker.cs
Comment (HamishBrownPFR) [inline modelCsharp/SVSModel/Configuration/DummyChecker.cs:1]: I think this file is redundant and can be deleted

File: modelCsharp/SVSModel/ModelInterface.cs
Comment (HamishBrownPFR) [inline modelCsharp/SVSModel/ModelInterface.cs:1]: Remove the debugging stuff from here (using System.Diagnostics  and Trace.Writeline...)

File: modelCsharp/TestModel/TestModel.sln
Comment (HamishBrownPFR) [inline modelCsharp/TestModel/TestModel.sln:1]: Is this file still needed, I thought we had added all the testing as projects into the SVSModel.sln

File: modelCsharp/TestModel/TestModel/TestModel.sln
Comment (HamishBrownPFR) [inline modelCsharp/TestModel/TestModel/TestModel.sln:1]: Is this .sln file needed, this has been added as a project to SVSModel.sln

File: modelCsharp/TestModel/testGraph/testGraph.sln
Comment (HamishBrownPFR) [inline modelCsharp/TestModel/testGraph/testGraph.sln:1]: Dont think we need this sln either

**Review by HamishBrownPFR — COMMENTED** [review]
few things to check and fix

File: modelCsharp/TestModel/TestModel/TestModel.sln
Comment (HamishBrownPFR) [reply modelCsharp/TestModel/TestModel/TestModel.sln:1]: @Stepania Did you miss this one or is it needed

Comment (Stepania) [reply modelCsharp/TestModel/TestModel/TestModel.sln:1]: I am going to check right now

**Review by HamishBrownPFR — APPROVED** [review]
